docs/the-new-architecture/use-app-template.md

```diff
@@ -1,7 +1,134 @@
 ---
 id: use-app-template
-title: Using the New App Template
+title: Creating a New Architecture App
 ---
 
-This section describes the new app template generated by the React Native CLI and how to use it properly.
-It should explain how to create a new app, which command should be issued to install the dependencies and to run it on the different platforms.
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+import RemoveGlobalCLI from '.././\_remove-global-cli.md';
+
+This page will help you create a new React Native app that uses the New Architecture.
+
+## Development environment
+
+Before continuing, make sure you've followed all the steps in [Setting up the
+development environment](getting-started.md), under the **React Native CLI
+Quickstart** tab.
+
+If following the setup guide, stop when you reach the section **Running your React Native
+Application**, and resume following this guide.
+
+**Note**: The New Architecture cannot currently be enabled in an Expo app.
+
+## Creating a new application
+
+<RemoveGlobalCLI />
+
+If you already have your development environment set up, create a new React Native project from the template:
+
+```shell
+npx react-native init AwesomeProject
+```
+
+If specifying a versioned template, use version 0.68 or later.
+
+## Configuration
+
+Follow the steps below to enable the New Architecture and build the app.
+
+### Enable Hermes
+
+Hermes is an open-source JavaScript engine optimized for React Native. [Hermes will be the default engine in the future](https://github.com/reactwg/react-native-new-architecture/discussions/4), and we highly recommend you use it.
+
+Please [follow the instructions on the React Native website](hermes.md) in order to enable Hermes in your application.
+
+### Enable the New Architecture
+
+#### Target OS
+
+<Tabs groupId="platform" defaultValue={constants.defaultPlatform} values={constants.platforms} className="pill-tabs">
+<TabItem value="ios">
+
+Navigate to the `ios` directory and run the following:
+
+```shell
+# from `ios` directory
+RCT_NEW_ARCH_ENABLED=1 bundle exec pod install
+```
+
+Then build and run the app as usual:
+
+```shell
+npx react-native run-ios
+```
+
+> **Tip**: You need to re-run `pod install` each time a dependency with native
```

Comment:
@cortinico: do we have any `:::`-box to wrap these tips nicely, instead of using quotes?

Reply:
Yes we do. I saw you folks already sorted this out, but in case you're wondering how this feature works for the future https://docusaurus.io/docs/markdown-features/admonitions